api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -92,19 +124,31 @@ This endpoint returns a list of budget subfunctions with their associated title
     + `2`
     + `3`
     + `4`
-+ budget_function: `050` (optional, string)
-+ budget_subfunction: `053` (optional, string)
 
-## BudgetFunctionResults (object)
-+ results (required, array[BudgetFunction], fixed-type)
+## AgencyData
++ cfo_agencies: (required, array[AgencyCFO])
++ other_agencies: (required, array[AgencyOtherAgencies])
++ federal_accounts: (required, array)
++ sub_agencies: (required, array)
+
+## AgencyCFO
++ name: `Department of Agriculture`
```

Comment:
Please include `optional` or `required` and data type for the data structure attributes